src/calendar/header/index.js

```diff
@@ -145,6 +154,41 @@ class CalendarHeader extends Component {
 
     const webProps = Platform.OS === 'web' ? {'aria-level': this.props.webAriaLevel} : {};
 
+    if (this.props.onPress || this.props.onLongPress) {
```

Comment:
Why the code duplication? You only need to replace a View with TouchableOpacity when there is an onPress / onLongPress prop and pass a few more props...